optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/queue/PartitionQueue.java

```diff
@@ -31,7 +31,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Thread-safe
+ * Thread-safe.
```

Comment:
According to javadoc guidelines, sentences should end with a dot and a sentance has at least 1 verb.
I'll change this into a sentence after merging :)